Tools/CISupport/build-webkit-org/steps.py

```diff
@@ -1944,6 +1945,24 @@ def run(self):
         return super().run()
 
 
+class CleanSaferCPPArchive(shell.ShellCommandNewStyle):
+    name = 'clean-safer-cpp-archive'
+    description = ['cleaning safer cpp archive']
+    descriptionDone = ['cleaned safer cpp archive']
+    haltOnFailure = False
+    flunkOnFailure = False
+    warnOnFailure = False
+
```

Comment:
You can consider adding __init__ and passing `logEnviron=False`. We do that consistently in ews, but not consistently in build.webkit.org. It's not critical though.